app/views/pages/_homepage-bottom.html.erb

```diff
@@ -1,33 +1,33 @@
 <div class="homepage-bottom">
   <div class="profile-title text-center" style="color: grey;">
-   <h3> Find Volunteers In 3 Steps</h3>
+   <h3> 有志者</h3>
```

Comment:
why is the translation for "find volunteers in 3 steps" now the name of the app?